internal/controller/vrg_kubeobjects.go

```diff
@@ -57,18 +57,15 @@ func kubeObjectsRecoverName(prefix string, groupNumber int) string {
 }
 
 func (v *VRGInstance) kubeObjectsProtectPrimary(result *ctrl.Result) {
-	v.kubeObjectsProtect(result, kubeObjectsCaptureStartConditionallyPrimary)
+	v.kubeObjectsProtect(result)
 }
 
 func (v *VRGInstance) kubeObjectsProtectSecondary(result *ctrl.Result) {
-	v.kubeObjectsProtect(result, kubeObjectsCaptureStartConditionallySecondary)
+	v.kubeObjectsProtect(result)
```

Comment:
If we remove the argument, kubeObjectsProtectPrimary() and kubeObjectsProtectSecondary() are the same, so we should drop them an call kubeObjectsProtect() directly.